conan/tools/microsoft/nmaketoolchain.py

```diff
@@ -0,0 +1,52 @@
+from conan.tools._compilers import cppstd_flag, build_type_flags
+from conan.tools.env import Environment
+
+
+class NMakeToolchain(object):
+    """
+    https://learn.microsoft.com/en-us/cpp/build/reference/running-nmake?view=msvc-170#toolsini-and-nmake
+    """
+    def __init__(self, conanfile):
+        """
+        :param conanfile: ``< ConanFile object >`` The current recipe object. Always use ``self``.
+        """
+        self._conanfile = conanfile
+        self._environment = None
+
+    @property
+    def cl_flags(self):
+        cppflags = []
+        build_type = self._conanfile.settings.get_safe("build_type")
+        if build_type in ['Release', 'RelWithDebInfo', 'MinSizeRel']:
+            cppflags.append("/DNDEBUG")
+
+        bt = build_type_flags(self._conanfile.settings)
+        if bt:
+            cppflags.extend(bt)
+
+        cppstd = cppstd_flag(self._conanfile.settings)
+        if cppstd:
+            cppflags.append(cppstd)
+        from conan.tools.microsoft import msvc_runtime_flag
+        flag = msvc_runtime_flag(self._conanfile)
+        if flag:
+            cppflags.append("-{}".format(flag))
+        return " ".join(cppflags).replace("-", "/")
+
+    @property
+    def environment(self):
+        # TODO: Seems we want to make this uniform, equal to other generators
+        if self._environment is None:
+            env = Environment()
+            env.append("CL", self.cl_flags)
+            env.append("UseEnv", "True")
```

Comment:
What is this for? I could not find any documentation about using this as an environment variable

Reply:
I could see that it could be part of msbuild command, e.g., `msbuild xxxxx /p:UseEnv=true`, but I'm not sure about its usage here 🤔

Reply:
Removed it, apparently it is not needed, better remove it and add it later if there is a case